test/dict.jl

```diff
@@ -900,7 +900,7 @@ end
 end
 
 @testset "Dict reduce merge" begin
-    f = (i::Vector{<:Dict}, o) -> begin
+    function f(i::Vector{<:Dict}, o)
```

Comment:
Should be `check_merge` ?

Reply:
I agree `check_merge` is better. I modeled the name `f` on a similar function in the file.